masonry/src/widgets/grid.rs

```diff
@@ -204,31 +218,35 @@ impl Grid {
         this.ctx.request_layout();
     }
 
-    #[expect(missing_docs, reason = "TODO")]
+    /// Set the spacing between grid items.
     pub fn set_spacing(this: &mut WidgetMut<'_, Self>, spacing: f64) {
         this.widget.grid_spacing = spacing;
         this.ctx.request_layout();
     }
 
-    #[expect(missing_docs, reason = "TODO")]
+    /// Set the number of columns of the grid.
```

Comment:
This makes it seem like the names of these functions are not so great given the wide difference between what the docs say and what a reasonable interpretation of the name of the function might have been.

Reply:
I can add a TODO, but otherwise I'd rather not go down that hole.